src/Razor/test/Microsoft.VisualStudio.Editor.Razor.Test/DefaultVisualStudioRazorParserIntegrationTest.cs

```diff
@@ -611,7 +611,7 @@ private async Task RunTypeKeywordTestAsync(string keyword)
         private static void DoWithTimeoutIfNotDebugging(Func<int, bool> withTimeout)
         {
 #if DEBUG
-            if (Debugger.IsAttached)
+            if (SystemDebugger.IsAttached)
```

Comment:
Seems like Roslyn added a Debugger namespace which confused this after updating, so I introduced an alias to keep things compiling.